### ISSUE TITLE
Using the mod in games with native support

### ISSUE THREAD

**Issue by Nooblet16** [Nukem9/dlssg-to-fsr3#131]
How do you go about doing this?
I know there were people who said the mod works in Immortals but that game has both DLSS FG and FSR3 FG. So you could basically force DLSS super resolution and FSR3 FG instead of relying on FSR for upscaling.

And that's the reason for this topic, would it be possible to get this to work on a game like Avatar that supports FSR3 FG but not DLSS FG (you have to mod that) ? It'd be nice if it was possible to use FSR3 FG in this game but with DLSS super resolution.

**Comment by ImASadBoy** [comment]
No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3

**Comment by Nooblet16** [comment]
> No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3

Yea I thought that'd be the way to do it indeed. Thanks

**Comment by tmatzxzone** [comment]
> No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3

im trying to do this on fallout 4's framegen by puredark but it keep crashing, managed to get the mod to popup but ctd on starting the game

**Comment by tmatzxzone** [comment]
> It works fine for me, are you sure you're moving the files in the game's .exe folder? Have you disabled the signature?

do u use enb or reshade? i think it might be my enb

**Comment by ImASadBoy** [comment]
> > It works fine for me, are you sure you're moving the files in the game's .exe folder? Have you disabled the signature?
> 
> do u use enb or reshade? i think it might be my enb

Reshade

**Comment by YaegerK** [comment]
> It works fine for me, are you sure you're moving the files in the game's .exe folder? Have you disabled the signature?

Is this the correct path?
Folder\game's exe
Folder\dxgi.dll
Folder\mods
Folder\nvngx.dll
Folder\dlssg_to_fsr3_amd_is_better.dll

I tried with TLOU and it works, but gameplay only, during cutscenes kinda works but usually when there's a cut the game freezes and crash for me. You can replicate this by playing the last bill's town cutscene, the game will crash when bill gives joel the tube or the suburbs when everyone go pass the staircase.

**Comment by ImASadBoy** [comment]
> > It works fine for me, are you sure you're moving the files in the game's .exe folder? Have you disabled the signature?
> 
> Is this the correct path? Folder\game's exe Folder\dxgi.dll Folder\mods Folder\nvngx.dll Folder\dlssg_to_fsr3_amd_is_better.dll
> 
> I tried with TLOU and it works, but gameplay only, during cutscenes kinda works but usually when there's a cut the game freezes and crash for me. You can replicate this by playing the last bill's town cutscene, the game will crash when bill gives joel the tube or the suburbs when everyone go pass the staircase.

I just moved them in the game's .exe folder for elden ring, then I installed the puredark mod following puredark's instructions and I was able to play without crashes.

**Comment by Nathan240600** [comment]
> No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3

Hello. Could you please elaborate on how to do this? I've been trying to mod Puredark FG and use this mod to change that into FSR3 but my game keep crashing. I'm testing with RDR2.

**Comment by ImASadBoy** [comment]
> > No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3
> 
> Hello. Could you please elaborate on how to do this? I've been trying to mod Puredark FG and use this mod to change that into FSR3 but my game keep crashing. I'm testing with RDR2.

![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/39035568/10615ecb-2b5d-4f15-ac9a-044389075d05)

I literally just moved the files to the game's .exe location, nothing more.

**Comment by Nathan240600** [comment]
> > > No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3
> > 
> > 
> > Hello. Could you please elaborate on how to do this? I've been trying to mod Puredark FG and use this mod to change that into FSR3 but my game keep crashing. I'm testing with RDR2.
> 
> ![image](https://private-user-images.githubusercontent.com/39035568/292243229-10615ecb-2b5d-4f15-ac9a-044389075d05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMxNzYxODIsIm5iZiI6MTcwMzE3NTg4MiwicGF0aCI6Ii8zOTAzNTU2OC8yOTIyNDMyMjktMTA2MTVlY2ItMmI1ZC00ZjE1LWFjOWEtMDQ0Mzg5MDc1ZDA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIxVDE2MjQ0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMyNDdjN2M2MThjZTNhODJlMGFkNjRiZmI4NWIxNmUyYzExZjIxYTBlNzRjMDY3NjlhNjZmMDJlYzliMWM2MzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.m8sUvTG1sYLQRHpDGZ-6abx9SQeanb3AoowgpX0iKlQ)
> 
> I literally just moved the files to the game's .exe location, nothing more.

Thanks. Probably an issue with Rockstar's launcher then...

**Comment by Pargonn** [comment]
same problem with BG3, someone knows how resolve?

**Comment by xxelitewarriorxx** [comment]
what about for gta v

**Comment by chikkibum** [comment]
> > No, you can use this mod with Avatars by using PureDark's mod that adds DLSSFG to the game, then using Nukem9 mod to change DLSSFG with FSR3
> 
> Hello. Could you please elaborate on how to do this? I've been trying to mod Puredark FG and use this mod to change that into FSR3 but my game keep crashing. I'm testing with RDR2.

did you finally find any solution ? I was trying to mod RDR2 but it keeps crashing for me

**Comment by Maxcension** [comment]
> same problem with BG3, someone knows how resolve?

Works for me. However, the HUD has huge ghosting issues (same with TLoU).

**Comment by Knell2231** [comment]
@tmatzxzone have you managed to get puredark fallout 4 framegen to work with nukem fsr3? im trying the same thing but it just wont work.

@ImASadBoy how exactly did you make nukem and puredark work with fallout 4?

**Comment by ImASadBoy** [comment]
> @tmatzxzone have you managed to get puredark fallout 4 framegen to work with nukem fsr3? im trying the same thing but it just wont work.
> 
> @ImASadBoy how exactly did you make nukem and puredark work with fallout 4?

First I followed Puredark's instructions for his mod, then i followed Nukem instructions for his mod (drag files in .exe folder)

**Comment by Knell2231** [comment]
> First I followed Puredark's instructions for his mod, then i followed Nukem instructions for his mod (drag files in .exe folder)

what specific version of nukem mod di you use? and did you use the dlsstweaks version? and i saw in one of the comments of puredark mod that he installed the mod directly into the root folder instead of using a mod manager?

**Comment by ImASadBoy** [comment]
> > First I followed Puredark's instructions for his mod, then i followed Nukem instructions for his mod (drag files in .exe folder)
> 
> what specific version of nukem mod di you use? and did you use the dlsstweaks version? and i saw in one of the comments of puredark mod that he installed the mod directly into the root folder instead of using a mod manager?

I used many versions, 0.90 works fine I tested that too, no dlsstweaks, normal method works fine, I didn't use any mod manager, I followed Puredark's instructions.

**Comment by Knell2231** [comment]
> I used many versions, 0.90 works fine I tested that too, no dlsstweaks, normal method works fine, I didn't use any mod manager, I followed Puredark's instructions.

I tried installing puredark's mod using my mod manager, it worked but the framegen (and toggling framegen on and off in the menu) wasnt doing anything and my general fps became lower.  nukem's dlssg was maybe working since a log file was being generated.

can you give me the specifics regarding puredark's instructions?

**Comment by neliavaal** [comment]
@ImASadBoy are you using an AMD gpu or nvidia?
